tidy3d/plugins/dispersion/fit_web.py

```diff
@@ -56,6 +64,26 @@ class AdvancedFitterParam(Tidy3dBaseModel):
         title="Number of inner iterations",
         description="Number of iterations in each inner optimization.",
     )
+    freeze_seeds: bool = Field(
+        True,
```

Comment:
we could also consider making this an `int` called `random_seed` with default value of `0` or something. That way the user can change the seed if they want.